docs/user/troubleshooting/plasma.md

```diff
@@ -81,11 +81,16 @@ ls -laH
 
 You should see the files you would usually see in your home directory when logged into Solus normally.
 
+:::caution Important
+
+The KDE project cautions users to not delete these directories: `~/.kde, ~/.kde4, ~/.local or ~/.config`.
```

Comment:
I'm not sure which wording is more correct, grammatically: "cautions users to not" or "cautions users not to." Thoughts?

Reply:
"to not install" is a split infinitive, though that's just a style thing. Emphasizing "not" with italics would make the sentence read more like how I would speak it aloud.

Reply:
It's better to remove the first part of the sentence ("The KDE project cautions users to"), it's just fluff and it makes the sentence difficult to read. 

Since the idea of this note is tell users to not delete the folders, it's better to keep it simple:

```suggestion
Do not delete the following directories: `~/.kde`, `~/.kde4`, `~/.local`, and `~/.config`.
```

Here's another alternative in case you don't want to use a negation (per the [Google style guide](https://developers.google.com/style/translation)):

```suggestion
Avoid deleting the following directories: `~/.kde`, `~/.kde4`, `~/.local`, and `~/.config`
```